grakn-migration/base/src/main/java/ai/grakn/migration/base/MigrationCLI.java

```diff
@@ -136,9 +136,9 @@ public static void printWholeCompletionMessage(MigrationOptions options){
             builder.append("Graph schema contains:\n");
             builder.append("\t ").append(graph.admin().getMetaEntityType().instances().count()).append(" entity types\n");
             builder.append("\t ").append(graph.admin().getMetaRelationType().instances().count()).append(" relation types\n");
-            builder.append("\t ").append("0 role types\n");
```

Comment:
haha! is this how we "fixed" this before? 😛

Reply:
🍌